Code/GraphMol/SmilesParse/CXSmilesOps.cpp

```diff
@@ -1938,6 +2020,93 @@ std::string get_bond_config_block(const ROMol &mol,
   return w + wU + wD;
 }
 
+std::string get_ringbond_cistrans_block(
+    const ROMol &mol, const std::vector<unsigned int> &atomOrder,
+    const std::vector<unsigned int> &bondOrder) {
+  if (!mol.getRingInfo()->isInitialized()) {
+    return "";
+  }
+
+  const auto rinfo = mol.getRingInfo();
+  std::string c = "", t = "", ctu = "";
+  for (unsigned int i = 0; i < bondOrder.size(); ++i) {
+    auto idx = bondOrder[i];
+    if (!rinfo->numBondRings(idx) ||
+        rinfo->minBondRingSize(idx) <
+            Chirality::minRingSizeForDoubleBondStereo) {
+      // we only do ring bonds of a minimum size
+      continue;
+    }
+    const auto bond = mol.getBondWithIdx(idx);
+    if (bond->getBondType() != Bond::BondType::DOUBLE &&
+        bond->getBondType() != Bond::BondType::AROMATIC) {
+      continue;
+    }
+    Bond::BondStereo bstereo = bond->getStereo();
+    if (bstereo != Bond::BondStereo::STEREOANY &&
+        bstereo != Bond::BondStereo::STEREOCIS &&
+        bstereo != Bond::BondStereo::STEREOTRANS) {
+      continue;
+    }
+
+    auto label = boost::str(boost::format("%d") % i);
```

Comment:
I guess I would have used either `std::to_string()` or something from `fmt`. - So I know in the future - why do you like boost::format?

Reply:
We use `boost::format` instead of something from `fmt` since we already have a dependency on boost and I generally try to add new dependencies.
Whenever we switch to C++20 I think we'll be able to start using `std::format` and simplify things further.

And as for not using `std::to_string()`: no good reason... I'm just not used to using it, but it definitely makes sense for these simple conversions.